paddle/scripts/paddle_build.bat

```diff
@@ -268,14 +268,21 @@ echo Build third_party successfully!
 
 set build_times=1
 :build_paddle
+:: reset clcache zero stats for collect PR's actual hit rate
+if "%CACHE_HIT_STATISTICS%"=="ON" clcache.exe -z
```

Comment:
I think CACHE_HIT_STATISTICS is not need, and it is WITH_CLCACHE, when WITH_CLCACHE=ON, collect_clcache_hits; WITH_CLCACHE=OFF: not collect_clcache_hits, or echo "clcache hit rate: 0%"

Reply:
It maybe fail when `clcache` command not exsits.

Reply:
can ``clcache --v`` adjuge first

Reply:
It is not necessary, `clcache` installed at first in script.